dev-tools/check_no_american_english.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+set -eu
+
+# Check for American English spellings that should be British English
+# Excludes: CHANGELOG.md (historical), vendor/, generated docs, class/method names, and test fixtures
+
+# @TODO v4: conclude on removing exceptions for "Analyzer" (shall be Analyser)
+
+american_spellings=$(
+    git grep -In \
+        -e '\bbehavior\b[^(]' \
+        -e '\bfavor\b[^(]' \
+        -e '\bhonor\b[^(]' \
+        -e '\banalyze\b[^(]' \
+        -e '\banalyzes\b[^(]' \
+        -e '\banalyzed\b[^(]' \
+        -e '\binitialize\b[^(]' \
+        -e '\binitializes\b[^(]' \
+        -e '\binitialized\b[^(]' \
+        ':!vendor/*' \
+        ':!*.lock' \
+    | grep -v 'behaviour' \
+    | grep -v 'favour' \
+    | grep -v 'honour' \
+    | grep -v 'analyse' \
+    | grep -v 'analyses' \
+    | grep -v 'analysed' \
+    | grep -v 'initialise' \
+    | grep -v 'initialises' \
+    | grep -v 'initialised' \
```

Comment:
Why do we need excludes for British here? Isn't it impossible to have these here after initial `git grep`?